src/improvements/script/simulate-verify-task.sh

```diff
@@ -8,46 +8,73 @@ simulate_verify_task() {
     root_dir=$(git rev-parse --show-toplevel)
     local_output_file="./simulation_output.txt"
     remote_output_file="./remote_output.txt"
+    forge_output_file="./forge_output.txt"
```

Comment:
Do we need all the file i/o here with local/remote/forge_output_file? When it fails for whatever reason before cleanup we leave these files lying around for devs to delete manually.

Reply:
We are scraping the output of other scripts to capture the payload and the hashes.

I tried to capture the output in a variable and scrape that, but we also want the output on console for debugging. It then becomes really hard to capture the output from both stdout, stderr in the variable and always show it regardless of the underlying script exit code.

The current system is more robust, as you always get all the output on the console for debugging. However, I've move the commands to remove files so that they are removed as soon as possible. If a script fails, only one file would remain, which also would have the relevant output if needed. The file would be removed in the next successful execution.

I hope that's a reasonable compromise.